Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -347,14 +334,18 @@ protected virtual void OnEscaping(EscapingEventArgs ev)
         /// Shows a message to the player when they pickup a custom item.
         /// </summary>
         /// <param name="player">The <see cref="Player"/> who will be shown the message.</param>
-        protected virtual void ShowMessage(Player player) => player.ShowHint($"You have picked up a {Name}\n{Description}", 10f);
+        protected virtual void ShowMessage(Player player)
+        {
+            player.ShowHint(string.Format(Instance.Config.PickedUpBroadcast.Content, Name, Description), Instance.Config.PickedUpBroadcast.Duration);
+        }
 
         /// <summary>
         /// Called when a player is given the item directly via a command or plugin.
         /// </summary>
         /// <param name="player">The <see cref="Player"/> who received the item.</param>
         protected virtual void ItemGiven(Player player)
```

Comment:
If you present it as an event, it'd be better to start the name with `On`, like `OnItemGiven`.

Reply:
That's what I'm going to do. Or pietro will, either way.

Reply:
Already removed
![Troll](https://user-images.githubusercontent.com/5269236/108890663-23f7e380-760e-11eb-8734-fc27669b36e8.gif)